packages/multicast_dns/lib/src/native_protocol_client.dart

```diff
@@ -42,7 +42,8 @@ class ResourceRecordCache {
     // Clear the cache for all name/type combinations to be updated.
     final Map<int, Set<String>> seenRecordTypes = <int, Set<String>>{};
     for (ResourceRecord record in records) {
-      seenRecordTypes[record.resourceRecordType] ??= {};
+      // TODO(dnfield): Update this to use set literal syntax when we're able to bump the SDK constraint.
+      seenRecordTypes[record.resourceRecordType] ??= {}; // ignore: prefer_collection_literals
```

Comment:
This should be `seenRecordTypes[record.resourceRecordType] ??= Set<String>(); // ignore: prefer_collection_literals`

Reply:
Done.